### PR TITLE
git: Ignore package-lock.json

### PR DESCRIPTION
**Pull request by svenschwermer**
Since this file seems to be generated by the barys script and it's not
supposed to be checked in, let's ignore it.

Signed-off-by: Sven Schwermer <sven.schwermer@disruptive-technologies.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 barys.log
 build/node_modules
 *.log
+build/package-lock.json
```
Comment (floion) [inline .gitignore]: I don't have this file generated in any of my repositories. @agherzan @ZubairLK do you get this created?

**Comment by svenschwermer** [comment]
Is there any update on this?

**Comment by floion** [comment]
I don't seem to have that file generated. What is the node version you use?

**Comment by floion** [comment]
Just checked with a newer version and indeed it gets generated.

**Comment by floion** [comment]
@balena-ci retest

**Comment by floion** [comment]
@svenschwermer please also add a `Change-type: patch` entry to your commit to make versionbot happy. And feel free to rebase also since you need to redo the PR

**Comment by floion** [comment]
@balena-ci retest

**Pull request merged**